packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.tsx

```diff
@@ -39,13 +33,33 @@ const getStyles = (theme: GrafanaTheme2) => ({
   disableGrow: css`
     width: 0%;
   `,
-  sortIcon: css`
-    position: absolute;
-    top: ${theme.spacing(1)};
+  header: css`
+    position: relative;
+    &,
+    & > button {
+      position: relative;
+      white-space: nowrap;
+      padding: ${theme.spacing(1)};
+    }
+    & > button {
```

Comment:
Is there a reason for the two different `& > button`? Specificity things?

Reply:
the first set of rules also applies to the `th`: `&, & > button`, while the second only to the button. there was an extra `position: relative;` at line 37 tho that i removed in 8aa4df9696